lib/ansible/modules/net_tools/basics/uri.py

```diff
@@ -58,9 +61,9 @@
   return_content:
     description:
       - Whether or not to return the body of the response as a "content" key in
-        the dictionary result. If the reported Content-type is
-        "application/json", then the JSON is additionally loaded into a key
-        called C(json) in the dictionary results.
+        the dictionary result.
+      - If the reported Content-type is "application/json", then the JSON is additionally loaded
+        into a key called C(json) in the dictionary results.
```

Comment:
It would really please me, if in the future, all of these unrelated changes could be kept out of PRs like this.  It exceedingly complicates reviewing.

Reply:
Well, I can't open a second PR with just the documentation changes unrelated to the PR because that would conflict with the changes needed for this PR and simply is too much work for any benefits.

Practically, if you don't think this is worth it, it would mean I won't be redoing docs at all.
IMO it's better as it is...

Reply:
When changes need to be made to docs that are in need of reformatting and/or cleanup, I've found it very helpful to perform all the cleanup first.

You can then open a PR for the cleanup and base your second PR on the first (if not waiting for the first to be merged).  Alternatively, if using a single PR, at least make the docs cleanup in a separate commit.

Doing either of those greatly simplifies the review process.

Reply:
Voila, the documentation changes are gone known. From a project's perspective if people spend time on improving something in their free time, I would be glad and accept it and not enforce too many rules too strictly...

BTW Not sending in any changes also simplifies the review process :-)